docs/openapi.yml

```diff
@@ -119,7 +119,7 @@ paths:
             text/html:
               schema:
                 type: string
-        400:
+        404:
```

Comment:
We still return 400 when a build can't be run. We need to add that status code here.